frontend/public/extend/devconsole/components/pipelines/PipelineVisualizationGraph.scss

```
@@ -0,0 +1,83 @@
+$border-color: var(--pf-global--BorderColor--light);
+$gutter: 1.8em;
+
+.odc-pipeline__graph {
+  background: white;
+  padding: 10px;
+  margin: 10px 0;
+  width: 80vw;
+  height: 20em;
+  overflow: auto;
+  position: relative;
+  -webkit-overflow-scrolling: touch;
+
+  // reset
+  & ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  & .odc-pipeline__stages {
```

If the top level block is specified correctly:
```suggestion
  &__stages {
```